src/main/java/com/google/sps/data/Recommended.java

```diff
@@ -0,0 +1,68 @@
+package com.google.sps.data;
+
+import java.util.*;
+
+public class Recommended {
+
+  public static Drink getDrink(String storeid, String drinkName) {
+    List<Drink> drinklist = DrinkDAO.getDrinksByStore(storeid);
+    for (Drink d : drinklist) {
+      if (d.getName().equals(drinkName)) {
+        return d;
+      }
+    }
+    return null;
+  }
+  // given a coffeeshop, return its tydrecommended score
+  public static double getScore(Drink d) {
+    double comments = d.getNumRatings();
+    // instead of shop rating get drink rating from this store
```

Comment:
My understanding is we don't have a concept of a "shop rating", correct? If yes, let's remove the reference to "shop rating" to avoid confusion.

Reply:
Removed it :)